components/AuthForm.tsx

```diff
@@ -66,78 +65,95 @@ export default function AuthForm() {
         await signupUser(email, password, firstName, lastName);
         navigateToVerifyPage();
       } catch (error) {
-        setAlertMessage(`Error while creating user: ${error}`);
-        setAlertSeverity("error");
-        setShowAlert(true);
+        toast({
+          title: "Error while creating user",
+          description: `${error}`,
+          status: "error",
+          isClosable: true,
+          duration: 6000,
+        });
       }
     }
   }
 
   return (
     <>
       <main className={styles.main}>
-        <h1>{isLoginMode ? "Login" : "Sign up"}</h1>
-        <form className="column" onSubmit={submitHandler}>
-          <TextField
-            required
-            label="Email"
-            id="email"
-            onChange={(event) => setEmail(event.target.value)}
-          />
-          <TextField
-            required
-            label="Password"
-            id="password"
-            type="password"
-            className="marginTopMedium"
-            onChange={(event) => setPassword(event.target.value)}
-          />
-          {!isLoginMode && (
-            <TextField
-              required
-              label="First name"
-              id="firstName"
-              className="marginTopMedium"
-              onChange={(event) => setFirstName(event.target.value)}
-            />
-          )}
-          {!isLoginMode && (
-            <TextField
-              required
-              label="Last name"
-              id="lastName"
-              className="marginTopMedium"
-              onChange={(event) => setLastName(event.target.value)}
-            />
-          )}
-          <Button variant="contained" type="submit" className="marginTopMedium">
-            {isLoginMode ? "login" : "create account"}
-          </Button>
-          <Button
-            variant="text"
-            type="button"
-            className="marginTopMedium"
-            onClick={() => switchLoginMode()}
-          >
-            {isLoginMode
-              ? "create new account"
-              : "log in with existing account"}
-          </Button>
-          {isLoginMode && (
-            <Button
-              variant="text"
-              type="button"
-              onClick={() => navigateToPasswordResetPage()}
-            >
-              Forgot Password
-            </Button>
-          )}
-        </form>
-        <CustomSnackbar
-          message={alertMessage}
-          severity={alertSeverity}
-          isOpenState={[showAlert, setShowAlert]}
-        />
+        <h1 className="text-3xl font-bold text-center">
+          {isLoginMode ? "Login" : "Sign up"}
+        </h1>
+        <div style={{width: 340}} className="mt-8">
```

Comment:
is this width limitation the best way to do it? shouldn't it fit the content and maybe only use a minimum width instead?

Reply:
min- or max-width depending on the needs is definitely a good idea! but that's something i would check with Jules or his design